src/test/java/stepdefs/FatimaTasks/SearchStepDefs.java

```diff
@@ -51,4 +49,22 @@ public void resultsRelevantToTheSearchTermAreReturned() {
       assertThat(header.getText()).as("Search results contains search term").contains("BBC");
     }
   }
+
+  @When ("I click on I'm Feeling Lucky")
+  public void clickOnImFeelingLucky() {
+    google.selectCookieButton();
+    driver.findElement(By.id("gbqfbb")).click();
+  }
+
+  @Then ("the url will reflect the search term")
+  public void verifyURLHasSearchTerm() {
+   driver.findElement(By.id("gbqfbb"));
+   String feelingLuckySearchText = String.valueOf(driver.findElement(By.id("gbqfbb")).getText());
```

Comment:
Could probably remove this line too if you complete the assertion mentioned below